test/parallel/test-http-agent-uninitialized-with-handle.js

```diff
@@ -13,11 +13,12 @@ socket._handle = {
   ref() { },
   readStart() { },
 };
-const req = new http.ClientRequest(`http://localhost:${common.PORT}/`);
 
 const server = http.createServer(common.mustCall((req, res) => {
   res.end();
-})).listen(common.PORT, common.mustCall(() => {
+})).listen(0, common.mustCall(() => {
+  const req = new http.ClientRequest(`http://localhost:${server.address().port}/`);
```

Comment:
Nit: long line I guess linter is not happy :)

Reply:
Linter probably forgives it because it contains a URL. But it should probably be wrapped.

Reply:
`make lint` runs with no errors -- nothing to do here I guess? I know the line is 83 chars long, is it worth it move the URL string to a new line?

Reply:
Seems okay to me